ubuntu/common/install_mpis.sh

```diff
@@ -20,14 +20,89 @@ tar -xvf $tarball
 mv $hpcx_folder $install_prefix
 hpcx_path=$install_prefix/$hpcx_folder
 
+# Module Files
+module_files_directory=/usr/share/modules/modulefiles/mpi
+mkdir -p $module_files_directory
+
+# HPC-X
+cat << EOF >> $module_files_directory/hpcx-$hpcx_version
+#%Module 1.0
+#
+#  HPCx $hpcx_version
```

Comment:
What's going on with the indenting here?

Reply:
This is a module file specification. Please refer - https://modules.readthedocs.io/en/stable/modulefile.html